From 8590657901d7cf0cf973dd297d2f1ca6d15610a9 Mon Sep 17 00:00:00 2001
From: Harry Youd <harry@harryyoud.co.uk>
Date: Mon, 29 Jan 2018 21:42:39 +0100
Subject: [PATCH 094/249] Adapt device_info template to new schema

Maintainers is now an empty array, so check for that

Change-Id: I491eac98f9082bde49678d5f612e7076084edabf
---
 _includes/deviceinfo.html          | 8 +-------
 _includes/templates/device_info.md | 6 ++----
 2 files changed, 3 insertions(+), 11 deletions(-)

diff --git a/_includes/deviceinfo.html b/_includes/deviceinfo.html
index 704f776..74e7a16 100644
--- a/_includes/deviceinfo.html
+++ b/_includes/deviceinfo.html
@@ -214,7 +214,7 @@ ul, ol {
     </tr>
     {%- endif %}
     <!-- lineage info -->
-    {% if device.maintainers %}
+    {% if device.maintainers != empty %}
     <tr>
         <td class="header" colspan="2">LineageOS info</td>
     </tr>
@@ -223,12 +223,6 @@ ul, ol {
         <td>
             <ul>
                 {% for el in device.maintainers %}
-                    {% if el == '' %}
-                        {% if device.maintainers | size == 0 %}
-                            <li>None</li>
-                        {% endif %}
-                        {% continue %}
-                    {% endif %}
                     <li>{{ el }}</li>
                 {% endfor %}
             </ul>
diff --git a/_includes/templates/device_info.md b/_includes/templates/device_info.md
index 81241cb..02fc014 100644
--- a/_includes/templates/device_info.md
+++ b/_includes/templates/device_info.md
@@ -1,13 +1,11 @@
 {% assign device = site.data.devices[page.device] %}
 {% include note_autogenerated.html %}
-{% unless device.maintainers %}
-{% if device.channels contains "discontinued" %}
+{% if device.maintainers == empty and device.channels contains "discontinued" %}
 {% include warning_discontinued_device.html %}
 {% endif %}
-{% endunless %}
 
 ## Get LineageOS for the {{ device.vendor }} {{ device.name }}
-{% if device.maintainers %}
+{% if device.maintainers != empty %}
 [Get the builds here](https://download.lineageos.org/{{ device.codename }})
 {% endif %}
 
-- 
2.17.1

